File: PHP/Booking/GetSubmittedBasketSummaryResult.php
<?php
namespace LTD_API;

class GetSubmittedBasketSummaryResult
{
    private $transactionReference;
    private $basketId;
    //instance of DeliveryInformation
    private $deliveryInformation;
    //array of BasketItem
    private $basketItems = array();
    private $bookingReference;
    //instance of DiscountInformation
    private $discountInformation;
    
    public function getTransactionReference()
    {
        return $this->transactionReference;
    }
    public function setTransactionReference($transactionReference)
    {
        $this->transactionReference = $transactionReference;
    }

    public function getBasketId()
    {
        return $this->basketId;
    }
    public function setBasketId($basketId)
    {
        $this->basketId = $basketId;
    }

    public function getDeliveryInformation()
    {
        return $this->deliveryInformation;
    }
    public function setDeliveryInformation($deliveryInformation)
    {
        $this->deliveryInformation = $deliveryInformation;
    }

    public function getBasketItems()
    {
        return $this->basketItems;
    }
    public function setBasketItems($basketItems)
    {
        $this->basketItems = $basketItems;
    }

    public function getBookingReference()
    {
        return $this->bookingReference;
    }
    public function setBookingReference($bookingReference)
    {
        $this->bookingReference = $bookingReference;
    }

    public function getDiscountInformation()
    {
        return $this->discountInformation;
    }
    public function setDiscountInformation($discountInformation)
    {
        $this->discountInformation = $discountInformation;
    }
}

File: Java/Booking/GetSubmittedBasketSummaryResult.java
package LTD_API;

import java.util.List;

public class GetSubmittedBasketSummaryResult
{
    private int transactionReference;
    private String basketId;
    private DeliveryInformation deliveryInformation;
    private List<BasketItem> basketItems;
    private String bookingReference;
    private DiscountInformation discountInformation;
    
    public int getTransactionReference()
    {
        return transactionReference;
    }
    public void setTransactionReference(int transactionReference)
    {
        this.transactionReference = transactionReference;
    }

    public String getBasketId()
    {
        return basketId;
    }
    public void setBasketId(String basketId)
    {
        this.basketId = basketId;
    }

    public DeliveryInformation getDeliveryInformation()
    {
        return deliveryInformation;
    }
    public void setDeliveryInformation(DeliveryInformation deliveryInformation)
    {
        this.deliveryInformation = deliveryInformation;
    }

    public List<BasketItem> getBasketItems()
    {
        return basketItems;
    }
    public void setBasketItems(List<BasketItem> basketItems)
    {
        this.basketItems = basketItems;
    }

    public String getBookingReference()
    {
        return bookingReference;
    }
    public void setBookingReference(String bookingReference)
    {
        this.bookingReference = bookingReference;
    }

    public DiscountInformation getDiscountInformation()
    {
        return discountInformation;
    }
    public void setDiscountInformation(DiscountInformation discountInformation)
    {
        this.discountInformation = discountInformation;
    }
}

